Improve README in terms of installation instructions and clarity (#1431)
roboflow/inference: inference_experimental/README.md

> curl -LsSf https://astral.sh/uv/install.sh | sh
> ```

Use the following command to install `inference-exp` on **CPU machine 💻** (below you can find more advanced options):

```bash
uv pip install inference-exp
# or - if you use pip
pip install inference-exp
```

<details>
<summary>👉 GPU installation</summary>

> As you mat learn in [📜 Principles and Assumptions](#-principles-and-assumptions), `inference-exp` is designed to 

Reviewer: ```suggestion
As you may learned from [📜 Principles and Assumptions](#-principles-and-assumptions), `inference-exp` is designed to 
```
